lib/streamlit/ReportSession.py

```diff
@@ -431,6 +433,29 @@ def handle_rerun_script_request(
 
         self.request_rerun(widget_state)
 
+    def handle_upload_file(self, upload_file=None):
```

Comment:
Why doss `upload_file` have a default value? The only place that calls `upload_file()` actually passes this argument, so the default is no needed (and confusing)

Reply:
I did it because all others methods called in server.py use default params. But You are right, I will remove it.